src/mscorlib/shared/System/TimeZoneInfo.Win32.cs

```diff
@@ -11,6 +11,20 @@
 using System.Threading;
 using Microsoft.Win32;
 
+#if CORERT
```

Comment:
There differences are not necessary. It is historical difference in conventions used for interop in CoreCLR and CoreRT. CoreFX interop conventions is what we are standardizing on for the stuff moved to the CoreLib shared partition.